### ISSUE TITLE
[Bug] Cookies not being passed when loading modules

### ISSUE THREAD

**Issue by hathvi** [grafana/grafana#9557]
> - What Grafana version are you using?

v4.7.0-pre1 (commit: 24a0eb9)

I went from v4.5.0 where panels were working fine to a v4.6.0-pre1 build containing commit d1c9760fa where I started experiencing the issue.

> - What OS are you running grafana on?

Official docker build with Docker running on Ubuntu 16.04

> - What did you do?

Attempted to load a dashboard using the `Worldmap Panel` and `Annotation Panel`

> - What was the expected result?

For the panels to load

> - What happened instead?

Grafana pops up an error saying 

```
Plugin Error
Fetch error: 403 Forbidden Instantiating https://**********/grafana/public/plugins/grafana-worldmap-panel/module.js Loading plugins/grafana-worldmap-panel/module
````
I'm proxying requests from oauth2_proxy to Grafana and have Grafana configured to use `auth.proxy`. The content of the 403 response shows my oauth2_proxy login page. Looking at the headers for the request to load the above URL I can see no cookies being passed with the request hence the 403 from my proxy server. Clicking the link in the console log message opens the link, passes the cookies, and displays the javascript content.

Here's the log as seen from the console

```
VM2319:1 GET https://********/grafana/public/plugins/grafana-worldmap-panel/module.js 403 (Forbidden)

Plugin component error Error: Fetch error: 403 Forbidden
  Instantiating https://********/grafana/public/plugins/grafana-worldmap-panel/module.js
  Loading plugins/grafana-worldmap-panel/module
    at index.js:33
    at <anonymous>
```

I'm not sure if this is intended and I'm expected to just allow unauthed requests to `/public`? I don't recall reading anything about this being a requirement in the Grafana documentation.

**Comment by torkelo** [comment]
duplicate of https://github.com/grafana/grafana/issues/9509

**Comment by torkelo** [comment]
fixed in v4.6.1 (releasing today)
